httpbin/helpers.go

```diff
@@ -433,3 +437,62 @@ func (b *base64Helper) Encode() ([]byte, error) {
 func (b *base64Helper) Decode() ([]byte, error) {
 	return base64.URLEncoding.DecodeString(b.data)
 }
+
+func wildCardToRegexp(pattern string) string {
+	components := strings.Split(pattern, "*")
+	if len(components) == 1 {
+		// if len is 1, there are no *'s, return exact match pattern
+		return "^" + pattern + "$"
+	}
+	var result strings.Builder
+	for i, literal := range components {
+
+		// Replace * with .*
+		if i > 0 {
+			result.WriteString(".*")
+		}
+
+		// Quote any regular expression meta characters in the
+		// literal text.
+		result.WriteString(regexp.QuoteMeta(literal))
+	}
+	return "^" + result.String() + "$"
+}
```

Comment:
I think we should simplify this a bit by only allowing a single trailing `*` as a wildcard.

Reply:
`wildCardToRegexp` is a very simple logic, if we want only allowing trailing `*`, we need add extra code.

in my opinion, there is no need to change the behavior

Reply:
My feeling is that the entire function would be much simpler if it only handled a single trailing wildcard, and that a single trailing wildcard will cover almost every real-world usecase. 

If you feel strongly about being able to match multiple wildcards, please add test coverage for
- a single wildcard in the middle of the string
- multiple wildcards in the string

Reply:
Yes, I have add some testcase about this func.